include/mxnet/ndarray.h

```diff
@@ -199,6 +199,11 @@ class NDArray {
     return ptr_->aux_shapes[i];
   }
 
+  const std::vector<TShape>& aux_shapes() const {
```

Comment:
Member functions defined inside classes are automatically inline. There is no need to put the key word `inline` in front of it.